src/index.js

```diff
@@ -258,6 +262,10 @@ function makeDailyDouble(player) {
   // console.log(dailyDouble)
   let highestPointClue = sortClues();
   wagerAmount = dailyDouble.determineWager(turns, player, highestPointClue);
+  console.log(dailyDouble)
+  console.log(turns)
+  console.log(highestPointClue)
+  console.log(wagerAmount)
```

Comment:
Be careful not to leave these in during submission (assuming they aren't needed for the purposes of your demo)